pkgs/development/libraries/qtutilities/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, qt5, cpp-utilities, cmake, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "qtutilities";
+  version = "5.13.0";
+
+  src = fetchFromGitHub {
+    owner = "Martchus";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03drxwpr3xdh4hm8pkd5mhjs0mzhz6634ldyq78pml39ciqm51nl";
+  };
+
+  buildInputs = [ qt5.qtbase cpp-utilities ];
```

Comment:
Now we just need to remove `qt5` references

Reply:
You mean changing it to just `qtbase`? This is what I figured from reading [section 9.15.1](https://nixos.org/nixpkgs/manual/#ssec-qt-libraries).

Reply:
Yep.

Reply:
But I get the `error: undefined variable 'qtbase' at ..`. What about the imports?

Reply:
You need to add `qtbase` and `qttools` to the formal function arguments at the top

Reply:
Got it, pushed an update.